vpc/proto/vpc.proto

```diff
@@ -148,6 +150,10 @@ message AssignIPResponseV3 {
         uint32 mtu = 2;
         Family family = 3;
     }
+    message Bandwidth {
+        uint64 bandwidth = 1;
+        uint64 burst = 2;
```

Comment:
Is this `bandwidth_limit` or `bandwidth_max` or something?

Reply:
What about "ceil"?